browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
+    "version": "3.0.1-beta",
+    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
+    "main": "./lib/index.js",
+    "license": "MIT",
+    "private": true,
+    "scripts": {
+        "build": "yarn build:go && yarn build:generate && yarn build:ts",
+        "build:ts": "node --max_old_space_size=4096 ./node_modules/.bin/tsc -b",
+        "build:generate": "go run ./scripts/generate_wasm_buffer.go",
+        "build:go": "GOOS=js GOARCH=wasm go build -o ./wasm/main.wasm ./go/main.go"
+    },
+    "dependencies": {
+        "@0x/order-utils": "^8.3.0",
+        "@0x/utils": "^4.5.0",
+        "@types/base64-arraybuffer": "^0.1.0",
+        "@types/node": "^12.7.2",
```

Comment:
Should these types be devDeps? Also, we typically set `@types/node` to `*` since there is a high chance that the end user is using a different version.

Reply:
Realized we don't need `@types/node` anymore so I just removed it. Also moved `typescript` and `@types/base64-arraybuffer` to `devDependencies`.